component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateUtil.java

```diff
@@ -382,20 +382,26 @@ private static boolean isUserExists(String userName, AuthenticationContext authe
     /**
      * Check whether user account is locked or not.
      *
-     * @param user Authenticated user.
+     * @param userName  user that passed from the cert CN value.
+     *  @param tenantDomain  from "authenticationcontex".
```

Comment:
I think authenticationcontex should be authenticationcontext.

Reply:
fixed